yaml-tests/src/test/resources/showcasing-tests.yamsql

```diff
@@ -168,6 +168,9 @@ test_block:
     # Like the options at the top of the file, a supported_version can be specified for
     # an individual test_block. This overrides the version specified in the file
     supported_version: 4.0.560.0
+    # specifies the connection options. This may not be supported by every connection type
+    connection_options:
+      DISABLE_PLANNER_REWRITING: false
```

Comment:
Is this just to make sure we can do this? Does this also mean that we end up skipping the whole file even though it's only this block that sets the connection option?